corehq/util/metrics/const.py

```diff
@@ -15,3 +15,9 @@
 MPM_LIVESUM = 'livesum'
 MPM_MAX = 'max'
 MPM_MIN = 'min'
+
+# Datadog tags
+MODULE_NAME_TAG = "module_name"
+
+# These tags will only be added for domains with FF DETAILED_TAGGING enabled
+GATED_DETAILED_TAGS = (MODULE_NAME_TAG)
```

Comment:
convert string to list (I assume this was meant to be a tuple but was missing a comma)
```suggestion
GATED_DETAILED_TAGS = [MODULE_NAME_TAG]
```

Reply:
yes it was, thanks. Ended up making it a set [1fd9571](https://github.com/dimagi/commcare-hq/pull/34226/commits/1fd9571e4777d2e43734248043e55e29f4768088)